### PR TITLE
Faster features

### PR DESCRIPTION
**Pull request by jospoortvliet**
The features page was annoyingly slow.

Instead of loading ~20 YouTube videos, now using jQuery to load them in a popup when you click them. 

It took me a day but the results:
* from 9.5 sec, 2.6mb to under 0.9 sec, 2mb in chromium
* from 20 sec, 27mb (!) to under 4 sec, 2mb in firefox

That's locally, we will have to see how it does on staging.

It looks differently, of course... But not too bad.

### CODE REVIEW

**Comment by jospoortvliet** [comment]
tested by @danimo too. Going for staging.

**Comment by phil-davis** [comment]
Very good. I have users in Nepal on 256, 384, 512Kbps connections. It is REALLY annoying when pages auto-download heaps of content when the user just wants to browse around and then click the content that they want to look at. And of course people with tablet/phones on mobile plans i an country who do not appreciate losing 20MB of their quota to browse to a page.

**Comment by jospoortvliet** [comment]
@phil-davis yeah, this will hopefully make it more feasible to check out our site in places like that. We're going to move to a new server too, supporting http2 and other stuff, should speed up things as well. And I'll look over some other pages to speed those up, too - most are fine but the YouTube stuff is big and heavy. Then again, I think pages with a single video are fine...

File: page-features.php [hunk 1]
```diff
@@ -269,3 +372,14 @@
 <p><a target="_blank" href="/contribute/" class="btn btn-lg btn-default"> <i class="icon-code"></i> Contribute to ownCloud</a></p>
 <!-- <p><a target="_blank" href="/conf/" class="btn btn-lg btn-default"> <i class="icon-group"></i> ownCloud Contributor Conf</a></p> -->
 </div>
+
+<!-- <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script> -->
+<link type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/themes/redmond/jquery-ui.css" rel="stylesheet" />
+<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js"></script>
```
Comment (LukasReschke) [inline page-features.php]: That won't work. Please use HTTPS urls.

**Pull request merged**